src/libcore/char/mod.rs

```diff
@@ -34,9 +34,7 @@ pub use self::convert::ParseCharError;
 pub use self::convert::{from_digit, from_u32};
 #[stable(feature = "decode_utf16", since = "1.9.0")]
 pub use self::decode::{decode_utf16, DecodeUtf16, DecodeUtf16Error};
-
-// unstable re-exports
-#[unstable(feature = "unicode_version", issue = "49726")]
+#[stable(feature = "unicode_version", since = "1.44.0")]
```

Comment:
Beta has branched to 1.44: https://github.com/rust-lang/rust/blob/beta/src/bootstrap/channel.rs#L15-L16

So this needs to be 1.45.0, same with the `const` item below. r=me with this.